docs/client-sdk/client-sdk-reference-guide.md

```diff
@@ -183,14 +182,14 @@ Table 1.	Prerequisites
 |--------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-|    Software                                      |    ·      Linux\* Ubuntu\* 16.04 / Ubuntu\* 18.04 using   OpenSSL\* 1.1.1f <br/>·      JDK11  <br/> ·      SDO release binaries package (for running test servers) <br/>     o The root directory of extracted binary package is referred to as <sdo_sdk_binaries> in subsequent sections. <br/>   ·      SDO Client SDK alpha-release binary package <br/>   o The root directory of extracted binary package is referred to as < SDOClientSDK> in subsequent sections. <br/>  ·      SDO Client SDK alpha-release sources package <br/>  o The root directory of extracted source code is referred to as < SDOClientSDK> in subsequent sections. <br/>  ·      Apache Maven\* (for building sample Java\* based test owner   service)   <br/>    Additional Ubuntu\* 16.04/18.04  prerequisites can be found in the   <releases source>/SDOClientSDK/README.md     |
+|    Software                                      |    ·      Linux\* Ubuntu\* 18.04 using   OpenSSL\* 1.1.1g <br/>·      JDK11  <br/> ·      SDO release binaries package (for running test servers) <br/>     o The root directory of extracted binary package is referred to as <sdo_sdk_binaries> in subsequent sections. <br/>   ·      SDO Client SDK alpha-release binary package <br/>   o The root directory of extracted binary package is referred to as < SDOClientSDK> in subsequent sections. <br/>  ·      SDO Client SDK alpha-release sources package <br/>  o The root directory of extracted source code is referred to as < SDOClientSDK> in subsequent sections. <br/>  ·      Apache Maven\* (for building sample Java\* based test owner   service)   <br/>    Additional Ubuntu\* 16.04/18.04  prerequisites can be found in the   <releases source>/SDOClientSDK/README.md     |
```

Comment:
Reference to Ubuntu 16.04 should be removed.

Additional Ubuntu\* 16.04/18.04 ==> Additional Ubuntu\* 18.04